### PR TITLE
chore(demo-playwright): rewrite combobox test to playwright

### PR DESCRIPTION
**Pull request by 40oleg**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [x] Tests related changes
- [ ] Documentation content changes

### CODE REVIEW

**Comment by 40oleg** [comment]
Previous PR: https://github.com/taiga-family/taiga-ui/pull/6064

**Comment by splincode** [comment]
@40oleg I see `Lint` job is failed

File: projects/demo-playwright/tests/kit/combo-box/combobox.spec.ts [hunk 1]
```diff
@@ -22,4 +23,101 @@ test.describe('ComboBox', () => {
         await input.click();
         await expect(page).toHaveScreenshot('01-combobox-dont-allow-disabled-04.png');
     });
+
+    describe('API', () => {
```
Comment (splincode) [inline projects/demo-playwright/tests/kit/combo-box/combobox.spec.ts:27]: test.describe

File: projects/demo-playwright/tests/kit/combo-box/combobox.spec.ts [hunk 2]
```diff
@@ -22,4 +23,101 @@ test.describe('ComboBox', () => {
...
+    });
 });
+
+async function visitBy(page: Page, strict: boolean): Promise<void> {
+    await tuiGoto(page, `components/combo-box/API?strict=${strict}&sandboxExpanded=true`);
+}
+
+async function focusWrapper(locator: Locator): Promise<void> {
+    await locator.click({force: true});
+}
```
Comment (splincode) [inline projects/demo-playwright/tests/kit/combo-box/combobox.spec.ts:123]: could you drop that utilities?

**Review by splincode — APPROVED** [review]

**Comment by splincode** [comment]
@nsbarsukov please, review

File: projects/demo-playwright/tests/kit/combo-box/combobox.spec.ts [hunk 3]
```diff
@@ -22,4 +23,99 @@ test.describe('ComboBox', () => {
...
+
+                await apiPageExample.click({force: true});
+
+                await expect(page).toHaveScreenshot(
+                    `correct-word-match-when-strict-focused-${strict}.png`,
+                );
+
+                await textfield.click();
+                await page.keyboard.press('Control+A');
+                await page.keyboard.press('Backspace');
```
Comment (nsbarsukov) [inline projects/demo-playwright/tests/kit/combo-box/combobox.spec.ts:107]: In playwright `press('Control+A')` does not work for mac os.
Could we use the following line instead ?

```suggestion
                await textfield.clear();
```

https://playwright.dev/docs/api/class-locator#locator-clear

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**